src/AccessibilityInsights.SharedUx/Enums/FileFilters.cs

```diff
@@ -7,11 +7,11 @@ namespace AccessibilityInsights.SharedUx.Enums
     /// </summary>
     public static class FileFilters
     {
-        public const string A11yFileFilter = "AccessibilityInsights files (*.a11ytest,*.a11yevent)|*.a11ytest;*.a11yevent|A11y Test files (*.a11ytest)|*.a11ytest|A11y Events files (*.a11yevent)|*.a11yevent|All files (*.*)|*.*";
-        public const string TestFileFilter = "AccessibilityInsights Test files (*.a11ytest)|*.a11ytest|All files (*.*)|*.*";
-        public const string EventsFileFilter = "AccessibilityInsights Events files (*.a11yevent)|*.a11yevent|All files (*.*)|*.*";
-        public const string TestExtension = ".a11ytest";
-        public const string EventsExtension = ".a11yevent";
```

Comment:
Is it the right move to pull these out into resource files? It seems like even if we localized, the file extensions themselves would remain unchanged.